### DISCUSSION TITLE
wayfire 0.8?

### DISCUSSION THREAD

**Discussion by bobblkabb** [void-linux/void-packages#47631] [Q&A] [answered]
It's been out for 2 months. 
Is anyone working on it? Are there any problems/difficulties?

Asking for a friend.

**Comment by classabbyamp** [comment, marked as answer]
doesn't appear anyone's done it yet.

**Comment by pondichys** [comment]
Wayfire 0.8.1 is available now. @bobblkabb would you please mark this as answered / closed ? Thank you.
